dev/com.ibm.ws.security.social_fat.LibertyOP.backchannelLogout/publish/files/serversettings/socialClient_configTests.xml

```diff
@@ -434,4 +436,220 @@
 	>
 	</oidcLogin>
 				
+	<oidcLogin
+		id="checkDupBcl_client1"
+		scope="profile email openid"
```

Comment:
That's the default value for this attribute.

Reply:
Removing this in a separate pull.